samples/snippets/src/main/java/dialogflow/cx/Example.java

```diff
@@ -62,6 +62,8 @@ public void service(HttpRequest request, HttpResponse response) throws Exception
             .getAsJsonObject();
     BufferedWriter writer = response.getWriter();
     writer.write(o.toString());
+    
+    System.out.println(o.toString());
```

Comment:
Needs a comment to explain what is happening

Reply:
this was just for testing